pose_collector:
  clock: 100
virtual_drone_clock: 100
field_cbf: [[-3, 3], [-3, 3]] # 衝突回避で避けられるよう、psiよりも一回り大きくとる
central_clock: 20
angle_aware:
  sigma: 0.2
  delta_decrease: 10
  gamma: 0.01
  alpha: 1
  # phi:
  #   range: [[-1, 1],[0, 1], [0,0.5], [0.78, 1.5707963], [-3.14159265358979,3.14159265358979]] #xyzvh, v:[pi/4, pi/2]
  #   density: [1, 1,0.5, 0.78, 6.28]  #v: pi/12, h: pi/12
  # psi:
  #   range: [[-1, 1],[0, 1]]
  #   density: [1, 1]
  # phi:
  #   range: [[-6, 6],[-6, 6], [0,0.5], [0.78, 1.5707963], [-3.14159265358979,3.14159265358979]] #xyzvh, v:[pi/4, pi/2]
  #   density: [0.1, 0.1, 0.5, 0.2617993878, 0.2617993878]  #v: pi/12, h: pi/12
  # psi:
  #   range: [[-8, 8],[-8, 8]]
  #   density: [0.1, 0.1]
  # phi:
  #   range: [[-1, 1],[-1, 1], [0,0.5], [0.78, 1.5707963], [-3.14159265358979,3.14159265358979]] #xyzvh, v:[pi/4, pi/2]
  #   density: [0.01, 0.01, 0.5, 0.2617993878, 0.2617993878]  #v: pi/12, h: pi/12
  # psi:
  #   range: [[-1.3, 1.3],[-1.3, 1.3]]
  #   density: [0.02, 0.02]
  # phi:
  #   range: [[0, 0.8],[0, 0.2], [0,0.5], [1.5707963, 1.5707963267948966], [-3.14159265358979,3.14159265358979]] #xyzvh, v:[pi/4, pi/2]
  #   density: [0.8, 0.8, 1, 2, 10]  #v: pi/12, h: pi/12
  #   r: 0.1
  phi:
    range: [[0, 0.8],[0, 0.2], [0,0.5], [0.7853981633974483, 1.5707963267948966], [-3.14159265358979,3.14159265358979]] #xyzvh, v:[pi/4, pi/2]
    density: [0.1, 0.1, 0.501, 0.19, 0.39]  #v: pi/12, h: pi/12
    r: 0.1
  
  
  psi:
    range: [[-1.1, 1.1], [-1.1, 0.1]]
    density: [0.05, 0.05]
    npy_name: psi_test.npy
  rviz_phi:
    range: [[-0.8, 0.8],[-0.8, -0.2], [0,0.5], [1.1780972450961724, 1.5707963], [-3.14159265358979,3.14159265358979]] #xyzvh, v:[pi/4, pi/2]
    density: [0.1, 0.1, 0.5, 0.009817477042468103, 0.2617993878]  #v: pi/12, h: pi/12
    r : 0.1
agents:
  agent_manager_clock: 20
  u_max: 0.3
  camera_sensor_rad: 0.7895822394
  kp_z: 1
  ref_z: 1
  kp_yaw: 1
  ref_yaw: 0
  camera_deg_range: [[-82, -5]]
  omega_max: 1
  camera_omega_max: 0.1
